### ISSUE TITLE
hiding and showing container, is freezing MixItUp

### ISSUE THREAD

**Issue by apfz** [patrickkunka/mixitup#234]
I have 3 different layouts for my items: tile, list and table. I use MixItUp for 'tile'.

When I perform a search query initially (from an input field) when the tile-view is active, filtering works great. 

Now switching to list,changing the query, is showing a filtered list (I have implemented it that way). Great. 

Switching back to tile and change the query, is suddenly not doing anything. The filter is passed the same way, mixitup is being triggered, but nothing is happening. 

In short, filtering while the container is not visible, seems to freeze MixItUp.

Is this a bug?

**Comment by patrickkunka** [comment]
Version 3 should resolve this issue. MixItUp it now a lot more robust when dealing with targets that are hidden by external code.

https://github.com/patrickkunka/mixitup/releases/tag/v3.0.0
